components/event-bus/cmd/event-bus-subscription-controller-knative/resources/manager.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: v1
```

Comment:
Please remove the deployment files from here, as it should be done later with helm charts.

Reply:
After discussios with the team, I moved them under `docs`.